docs/csharp/language-reference/operators/delegate-operator.md

```diff
@@ -12,12 +12,11 @@ The `delegate` operator creates an anonymous method that can be converted to a d
 
 [!code-csharp-interactive[anonymous method](snippets/shared/DelegateOperator.cs#AnonymousMethod)]
 
-> [!NOTE]
-> Beginning with C# 3, lambda expressions provide a more concise and expressive way to create an anonymous function. Use the [=> operator](lambda-operator.md) to construct a lambda expression:
->
-> [!code-csharp-interactive[lambda expression](snippets/shared/DelegateOperator.cs#Lambda)]
->
-> For more information about features of lambda expressions, for example, capturing outer variables, see [Lambda expressions](lambda-expressions.md).
+Lambda expressions provide a more concise and expressive way to create an anonymous function. Use the [=> operator](lambda-operator.md) to construct a lambda expression:
+
+[!code-csharp-interactive[lambda expression](snippets/shared/DelegateOperator.cs#Lambda)]
+
+For more information about features of lambda expressions, for example, capturing outer variables, see [Lambda expressions](lambda-expressions.md).
```

Comment:
I think it's still good to keep the NOTE block to highlight the lambda expressions.